src/aboutPage/AboutPageEn.tsx

```diff
@@ -0,0 +1,214 @@
+import React, { Fragment, ReactElement } from 'react';
+
+import { Link } from '../common/copyOfHDSLink/Link';
+import FocusableH1 from '../common/focusableH1/FocusableH1';
+
+function AboutPageEn(): ReactElement {
+  return (
+    <Fragment>
+      <FocusableH1>About Helsinki profile</FocusableH1>
+      <p>
+        The Helsinki Profile is a user ID and storage service for personal and
+        contact information. It brings together the customer&apos;s personal
+        data and contacts with different city services in one place.
+      </p>
+      <p>
+        The Helsinki Profile and its authentication to various digital
+        applications
+        <ul>
+          <li>strongly via suomi.fi authentication</li>
+          <li>
+            light by using the Helsinki ID email address + password combination
+          </li>
+        </ul>
+      </p>
+      <p>
+        The use of the Helsinki profile and the data stored in it is mainly
+        based on the customer&apos;s consent for light authentication and on a
+        legal basis for strong authentication.{' '}
+      </p>
+      <p>
+        The Helsinki profile supports Single Sign-On (SSO). This means that a
+        customer who is already logged in to one service can switch to another
+        without having to log in again, and the same contact details will be
+        available to both services. The digital service using the Helsinki
+        profile determines the level of authentication required from the
+        customer: light or strong.
+      </p>
+      <h2>Strong identification</h2>
+      <p>
+        If the use of the service requires official personal data, the customer
+        is offered strong authentication to the profile via Suomi.fi&apos;s
+        central authentication service so that official personal data can be
+        transmitted to the service.
+      </p>
+      <p>
+        As official personal data, the Helsinki profile stores the basic
+        information about the Customer obtained through Suomi.fi from the
+        Population Information System, which includes
+        <ul>
```

Comment:
I noticed that there is a console warning: `<ul> cannot appear as a descendant of <p>`. Should be an easy fix 🤔